internal/apischema/gcp/gcp.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Envoy AI Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package gcp
+
+import "google.golang.org/genai"
+
+type GenerateContentRequest struct {
```

Comment:
A you help add the reference links where these types are from and comments to the types.